linux_os/guide/system/accounts/accounts-banners/gui_login_banner/gui_login_dod_acknowledgement/oval/shared.xml

```diff
@@ -0,0 +1,49 @@
+<def-group>
+  <definition class="compliance" id="gui_login_dod_acknowledgement" version="1">
+    <metadata>
+      <title>The SUSE operating system must display the Standard Mandatory DoD Notice and Consent Banner until users acknowledge the usage conditions and take explicit actions to log on for further access to the local graphical user interface (GUI).</title>
```

Comment:
Please use the [oval_metadata macro](https://complianceascode.readthedocs.io/en/latest/jinja_macros/oval.html?highlight=oval_metadata#template-oval_metadata)